Assets/HoloToolkit/Input/Scripts/Microphone/MicStream.cs

```diff
@@ -62,29 +68,32 @@ public static int MicStartStream(bool keepData, bool previewOnDevice)
 		/// Shuts down the connection to the microphone. Data will not longer be received from the microphone.
 		/// </summary>
 		/// <returns>error code or 0</returns>
-		[DllImport("MicStreamSelector")]    public static extern int MicStopStream();
+		[DllImport("MicStreamSelector")]
+        public static extern int MicStopStream();
 
 		/// <summary>
 		/// Begins recording microphone data to the specified file.
 		/// </summary>
 		/// <param name="filename">The file will be saved to this name. Specify only the wav file's name with extensions, aka "myfile.wav", not full path</param>
 		/// <param name="previewOnDevice">If true, will play micstream in speakers</param>
 		/// <returns></returns>
-		[DllImport("MicStreamSelector")]    public static extern int MicStartRecording(string filename, bool previewOnDevice);
+		[DllImport("MicStreamSelector")]
+        public static extern int MicStartRecording(string filename, bool previewOnDevice);
 
 		/// <summary>
 		/// Finishes writing the file recording started with MicStartRecording.
 		/// </summary>
 		/// <param name="sb">returns the full path to the recorded audio file</param>
-        [DllImport("MicStreamSelector")]    public static extern void MicStopRecording(StringBuilder sb);
+        [DllImport("MicStreamSelector")]
+        public static extern void MicStopRecording(StringBuilder sb);
 
 		/// <summary>
 		/// Finishes writing the file recording started with MicStartRecording.
 		/// </summary>
 		/// <returns>the full path to the recorded audio file</returns>
 		public static string MicStopRecording()
 		{
-			StringBuilder builder = new StringBuilder(1024);
+			StringBuilder builder = new StringBuilder(MAX_PATH); 
 			MicStopRecording(builder);
```

Comment:
Nit: Reminding myself to refactor the unmanaged code to eliminate a lot of these warts.